build/Dockerfile

```diff
@@ -1,35 +1,27 @@
 FROM quay.io/app-sre/boilerplate:image-v3.0.3 AS builder
-FROM golang:1.19 as builder
-ARG TARGETOS
-ARG TARGETARCH
 
-WORKDIR /workspace
-# Copy the Go Modules manifests
+RUN mkdir -p /workdir
```

Comment:
```suggestion
```
WORKDIR will make a directory if it doesn't exist

Reply:
This is all your fault....